### PR TITLE
Update envoy sha

### PR DESCRIPTION
**Pull request by bianpengyuan**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by Shikugawa** [comment]
@bianpengyuan I'm thinking that this update should be conducted after merged this. https://github.com/envoyproxy/envoy/pull/12042

File: WORKSPACE [hunk 1]
```diff
@@ -37,10 +37,10 @@ bind(
 # 1. Determine SHA256 `wget https://github.com/envoyproxy/envoy-wasm/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
 # 2. Update .bazelversion, envoy.bazelrc and .bazelrc if needed.
 #
-# Commit time: 6/30/20
-ENVOY_SHA = "06d236f62097bebc3af7b4f3154b7beca73f1339"
+# Commit time: 7/13/20
```
Comment (shamsher31) [inline WORKSPACE:40]: nit: This should be `Commit time` or `Commit date`? Because there is no time mentioned on the date.

Comment (bianpengyuan) [reply WORKSPACE]: Done.

**Comment by bianpengyuan** [comment]
@Shikugawa trying to update the sha with that commit and remove the absl patch, but seems like the absl has not yet been updated to include the new CPU type?

I will merge this without that PR since I think there are people waiting for this, we can always update the sha again later.

**Pull request merged**